### ISSUE TITLE
ufo export path

### ISSUE THREAD

**Issue by schriftgestalt** [jpt/font-scripts#1]
https://github.com/jpt/font-scripts/blob/bf6ab84cda83433860719f37ea50785b7d2f1593/Glyphs/Export%20Designspace.py#L90

I would not rely on the ufo export but export the .ufos myself from within the script. And have you tried [glyphslib](http://github.com/googlefonts/glyphsLib)? That can write a designspace file from a .glyphs file and back.

**Comment by jpt** [comment]
Makes sense RE: exporting UFOs. It will be necessary anyway if people use brace/bracket layers. 

Did not know about glyphslib - nice! I guess I should just turn this script into a UI for it :)

**Comment by schriftgestalt** [comment]
I tried it some time ago to run glyphsLib from within Glyphs. The API should allow it. But it might need some tweaking.
